external_example/integration_tests/CMakeLists.txt

```diff
@@ -1,25 +1,16 @@
-#set(_libs
-#    dronecore
-#    pthread
-#)
-
-list(APPEND integration_tests
-    hello_world
+add_executable(yuneec_integration_tests_runner
```

Comment:
should this be "yuneec" - very vendor specific!

Reply:
Oops, thx.